api/filters/authn/required.go

```diff
@@ -28,7 +29,7 @@ func (raf *RequiredAuthnFilter) Run(request *web.Request, next web.Handler) (*we
 	ctx := request.Context()
 	if _, ok := web.UserFromContext(ctx); !ok {
 		log.C(ctx).Error("No authenticated user found in request context during execution of filter ", raf.Name())
-		return nil, errUnauthorized
+		return nil, middlewares.UnauthorizedHTTPError(nil)
```

Comment:
doesn't seem right to be under middlewares. security.UnauthorizedHttpError seems better